modules/sdk/gradle-plugins-node/CHANGELOG.markdown

```diff
@@ -268,6 +268,14 @@ task's `useNpmCI` property is set to `true`.
 beginning from version 6.2.1. Download and install Node.js and NPM separately
 for Node.js versions 5.5.0 - 6.2.0.
 
+## 4.5.0 - 2018-11-16
+
+### Added
+- [LPS-87465]: Add the property `production` to all tasks that extend
+`ExecuteNodeTask`. If `true`, devDependencies are not installed when running
+`npm install` without any arguments and sets `NODE_ENV=production` for lifecycle
+scripts.
```

Comment:
@petershin Is `devDependencies` in this description some sort of property? Or are we referring to dependencies in general here?

Reply:
https://docs.npmjs.com/files/package.json#devdependencies

Reply:
Thanks, Dave!